.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'Please replace with a clear and descriptive title'
+labels: 'type: bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Configuration**
+
+- If possible, please copy/paste below your `netlify.toml`.
+- Please enter the following command in a terminal and copy/paste its output:
+
+```bash
+npx envinfo --system --binaries --npmPackages netlify-cli --npmGlobalPackages netlify-cli
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
Do many people submit screenshots/do you want them?

Reply:
Changed to `CLI Output` here https://github.com/netlify/cli/pull/1090/files#diff-74c2649c999338ed7a2833d79d4b25c3R36